### PR TITLE
Swashbuckle project and IOperationFilter issue-#6

### PR DESCRIPTION
**Pull request by berkayakcay**
- Init Swashbuckle project
- Added reference of _Swashbuckle.AspNetCore.SwaggerGen_
- Added _Swashbuckle.AspNetCore.SwaggerGen.**IOperationFilter**_ 
---
@GeorgDangl
waiting for _NSwag_ project structure and test examples then i will have been happy to made Swashbuckle side completed.

issue-#6

### CODE REVIEW

File: src/LightQuery.Swashbuckle/LightQueryOperationFilter.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+{
+    /// <summary>
+    /// Generates operation filter for LightQuery's paramters
+    /// Swashbuckle <see cref="IOperationFilter"/>
+    /// </summary>
+    /// <remarks>OpenAPI Support</remarks>
+    public class LightQueryOperationFilter : IOperationFilter
+    {
+        /*
+         * TODO : how can open api schema be parametric?
```
Comment (berkayakcay) [inline src/LightQuery.Swashbuckle/LightQueryOperationFilter.cs:19]: @GeorgDangl I don't know but, People may want to localize or customize **Description** or **Schema Example**

Comment (GeorgDangl) [reply src/LightQuery.Swashbuckle/LightQueryOperationFilter.cs:19]: I wouldn't wory about localization right now, I think English is fine so far.
If required, developers may hook into the `PostProcess` to alter the schema as they wish.

**Comment by GeorgDangl** [comment]
Hey @berkayakcay, I've just pushed the changes for NSwag to the dev branch: d1643f221ace831145bdc7d4c54d409cd745383d

File: src/LightQuery.Swashbuckle/LightQuery.Swashbuckle.csproj [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        <Description>Extensions to use LightQuery with Swashbuckle</Description>
+        <Copyright>(c) $([System.DateTime]::Now.Year) Georg Dangl</Copyright>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <PackageProjectUrl>https://github.com/GeorgDangl/LightQuery</PackageProjectUrl>
+        <RepositoryUrl>https://github.com/GeorgDangl/LightQuery.git</RepositoryUrl>
+        <RepositoryType>git</RepositoryType>
+        <PackageTags>Asp-Net-Core Querying Sorting Filtering</PackageTags>
+        <PackageIcon>gd_icon_256.png</PackageIcon>
+        <SignAssembly>true</SignAssembly>
+        <!--        <AssemblyOriginatorKeyFile>LightQuery.Swashbuckle.snk</AssemblyOriginatorKeyFile>-->
```
Comment (berkayakcay) [inline src/LightQuery.Swashbuckle/LightQuery.Swashbuckle.csproj:17]: @GeorgDangl need help for snk

File: test/LightQuery.IntegrationTestsServer/Controllers/AsyncLightQueryController.cs [hunk 3]
```diff
@@ -15,6 +15,7 @@ public AsyncLightQueryController(LightQueryContext context)
 
         private readonly LightQueryContext _context;
 
+        [HttpGet]
```
Comment (berkayakcay) [inline test/LightQuery.IntegrationTestsServer/Controllers/AsyncLightQueryController.cs:18]: Added for Swashbuckle obligations

File: test/LightQuery.IntegrationTestsServer/Controllers/AsyncPaginatedLightQueryController.cs [hunk 4]
```diff
@@ -16,6 +16,7 @@ public AsyncPaginatedLightQueryController(LightQueryContext context)
 
         private readonly LightQueryContext _context;
 
+        [HttpGet]
```
Comment (berkayakcay) [inline test/LightQuery.IntegrationTestsServer/Controllers/AsyncPaginatedLightQueryController.cs:19]: Added for Swashbuckle obligations

File: test/LightQuery.IntegrationTestsServer/Controllers/LightQueryController.cs [hunk 5]
```diff
@@ -14,6 +14,7 @@ public LightQueryController(LightQueryContext context)
 
         private readonly LightQueryContext _context;
 
+        [HttpGet]
```
Comment (berkayakcay) [inline test/LightQuery.IntegrationTestsServer/Controllers/LightQueryController.cs:17]: Added for Swashbuckle obligations

File: test/LightQuery.IntegrationTestsServer/Controllers/PaginatedLightQueryController.cs [hunk 6]
```diff
@@ -15,6 +15,7 @@ public PaginatedLightQueryController(LightQueryContext context)
 
         private readonly LightQueryContext _context;
 
+        [HttpGet]
```
Comment (berkayakcay) [inline test/LightQuery.IntegrationTestsServer/Controllers/PaginatedLightQueryController.cs:18]: Added for Swashbuckle obligations

File: test/LightQuery.IntegrationTestsServer/Controllers/ValuesController.cs [hunk 7]
```diff
@@ -14,6 +14,7 @@ public ValuesController(LightQueryContext context)
 
         private readonly LightQueryContext _context;
 
+        [HttpGet]
```
Comment (berkayakcay) [inline test/LightQuery.IntegrationTestsServer/Controllers/ValuesController.cs:17]: Added for Swashbuckle obligations

**Pull request merged**